### PR TITLE
feat(styling): setup Tailwind CSS

### PR DESCRIPTION
**Pull request by mohamilin**
## Description

- Setup tailwindcss in project santrihub

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -13,6 +13,15 @@
...
       ]
+    },
+    {
+      "login": "mohamilin",
+      "name": "Moh Amilin",
+      "avatar_url": "https://avatars.githubusercontent.com/u/50976240?v=4",
+      "profile": "https://www.linkedin.com/in/moh-amilin-6a5015128/",
+      "contributions": [
+        "doc"
+      ]
```
Comment (zainfathoni) [inline .all-contributorsrc:24]: Please revert the changes to this file. We shouldn't manually update this file.

We should use GitHub comments to add your name, like what I did here: https://github.com/SantriHub/app/pull/3#issuecomment-1153064029.
You can read more about it over here: https://allcontributors.org/docs/en/bot/usage

**Review by zainfathoni — CHANGES_REQUESTED** [review]
Thanks for initiatively making this PR. 🙏 
Overall, it looks good, but I have minor suggestions for this PR.

File: package.json [hunk 2]
```diff
@@ -2,16 +2,27 @@
...
   "sideEffects": false,
   "scripts": {
-    "build": "remix build",
-    "deploy": "deployctl deploy --prod --include=build,public --project=<your deno deploy project> ./build/index.js",
+    "generate:css": "npx tailwindcss -o ./app/tailwind.css",
+    "build": "run-s build:*",
+    "build:fetch": "npm run fetch",
+    "build:css": "npm run generate:css -- --minify",
+    "build:remix": "remix build",
+    "deploy": "deployctl deploy --prod --include=build,public --project=senarai ./build/index.js",
```
[the remark this answers is not in the document]
Comment (zainfathoni) [reply package.json]: Btw, I just added the automatic deployment at #8, and it's now deployed at https://santrihub.deno.dev.

File: .all-contributorsrc (hunk 1 above)
Comment (mohamilin) [reply .all-contributorsrc]: Oke, thnks mas @zainfathoni , updated the commits

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (mohamilin) [reply package.json]: oke, thnks mas @zainfathoni

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@ node_modules
 /build
 /public/build
 .env
+/app/tailwind.css
```
Comment (zainfathoni) [inline .gitignore]:
```suggestion
/app/tailwind.css

```

**Review by zainfathoni — APPROVED** [review]
LGTM 💯
Thanks for making the requested changes! 🙏

**Pull request merged**